drivers/network/dd/nvnet/netnv.inf

```diff
@@ -262,7 +256,6 @@ PP = "Packet Priority"
 VT = "VLAN Tagging"
 VI = "VLAN ID"
 JF = "Jumbo Frame"
-
 nForce.DeviceDesc = "nVidia nForce PCI Ethernet Controller"
 nForce2.DeviceDesc = "nVidia nForce2 PCI Ethernet Controller"
 nForce3.DeviceDesc = "nVidia nForce3 PCI Ethernet Controller"
```

Comment:
There was an empty line which separated logically different content.

Reply:
No, if I added this one back, then you wouldn't know that "Localizable" is still the active comment for that section. And that is the differentiator here. Open the file in an editor and you will understand.

Reply:
`; Localizable` comment marks everything below it as localizable, unless another comment follows, marking it as something else. Empty lines are not related to that in any way.

Reply:
That makes no sense, because the same could be said about each and every multi-line-spanning comment. There is no need to separate anything within the localized strings section.